scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -1489,7 +1489,7 @@ public boolean changeStartAt(JobId jobId, String startAt) {
     }
 
     @Override
-    public Job getJobContent(JobId jobId) throws UnknownJobException, NotConnectedException, PermissionException {
+    public String getJobContent(JobId jobId) throws UnknownJobException, NotConnectedException, PermissionException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

Reply:
I think sonarqube means that there is one exception in the throws declaration which is not actually thrown, can you check that?